src/mmw/js/src/draw/tests.js

```diff
@@ -26,6 +26,20 @@ var sandboxId = 'sandbox',
           [-75.16287803649902, 39.95161218948083],
           [-75.16518473625183, 39.95194939669509],
           [-75.16472339630127, 39.953446247674904]]]]
+    },
+    TEST_POLYGON = {
```

Comment:
Because `TEST_POLYGON` is defined but never used, it is causing linting errors and the build to be marked as UNSTABLE. You can run the linter locally with:

```
$ ./scripts/npm.sh run lint

+ ARGS='run lint'
+ vagrant ssh app -c 'cd /opt/app && envdir /etc/mmw.d/env npm run lint'

> model-my-watershed@0.0.1 lint /opt/app
> jshint $(find js -name *.js -not -path '*/shim/*')

js/src/draw/tests.js: line 30, col 5, 'TEST_POLYGON' is defined but never used.

1 error
```

Then there's a bunch of npm nonsense after it, but that's the important bit.